### PR TITLE
rdf action should be able to return the raw-rdf values

### PR DESCRIPTION
**Pull request by DrDomenicoMarson**
Hello everyone, 

I just updated the rdf action with an optional boolean argument `raw_rdf`.
When `raw_rdf` is set to True, the returned rdf values are not normalized, as the `rawrdf` argument in cpptraj.

### CODE REVIEW

**Comment by hainm** [comment]
hi @DrDomenicoMarson 

Thank you for your PR. Can you please add a quick test for raw_rdf to this file? https://github.com/Amber-MD/pytraj/blob/master/tests/test_analysis/test_rdf.py

File: pytraj/all_actions.py [hunk 1]
```diff
@@ -1171,13 +1174,14 @@ def rdf(traj=None,
     center1_ = 'center1' if center_solvent else ''
     center2_ = 'center2' if center_solute else ''
     nointramol_ = 'nointramol' if not intramol else ''
+    raw_rdf_ = "rawrdf pytraj_tmp_output_raw.agr" if raw_rdf else ''
```
[the remark this answers is not in the document]
Comment (DrDomenicoMarson) [reply pytraj/all_actions.py:1177]: I don't fully understand the interaction with the cpptraj library, as I'm not familiar with python/libraries interfaces, but I tried to run a version without the filename and it doesn't work.

The cpptraj command needs to have both the keyword and the file name where to write the data, so I suspect that both are needed also here!

[the remark this answers is not in the document]
Comment (hainm) [reply pytraj/all_actions.py:1177]: I see.

**Comment by hainm** [comment]
@DrDomenicoMarson  Please 
```
git fetch origin
git merge origin/master
```

to your branch to fix the failed build.

**Comment by DrDomenicoMarson** [comment]
Thanks @hainm for the help!

I added the needed test, do you think this PR will be merged into the upcoming AMBER24 release? To know if I'll be able to switch back to the plain AMBER version of pytraj :-)

Also, I made this PR focusing on maintaining consistent behavior in the pt.rdf() function, that is, one single dataset is always returned. However, when invoked with the rawrdf argument, cpptraj will produce both the raw and normalized data. Now, if the user is interested in both datasets, he/she has to make two pt.rdf() calls.

It would be easy to make pt.rdf() return both the datasets, when called with the raw_rdf argument, but I don't know what is your opinion on having a function that returns a variable number of datasets based on the arguments... please, let me know what you think about it.

**Comment by hainm** [comment]
> do you think this PR will be merged into the upcoming AMBER24 release? To know if I'll be able to switch back to the plain AMBER version of pytraj :-)

@DrDomenicoMarson
yeah, the PR will be merged to the upcoming release of AMBER/AmberTools

**Comment by hainm** [comment]
> Also, I made this PR focusing on maintaining consistent behavior in the pt.rdf() function, that is, one single dataset is always returned. However, when invoked with the rawrdf argument, cpptraj will produce both the raw and normalized data. Now, if the user is interested in both datasets, he/she has to make two pt.rdf() calls.
> 
> It would be easy to make pt.rdf() return both the datasets, when called with the raw_rdf argument, but I don't know what is your opinion on having a function that returns a variable number of datasets based on the arguments... please, let me know what you think about it.

I think let's make the return value simple for now (keep the consistent behavior). Thanks.

**Comment by hainm** [comment]
Thanks @DrDomenicoMarson very much!

**Comment by DrDomenicoMarson** [comment]
> Thanks @DrDomenicoMarson very much!

THanks for the assistance :-)

**Pull request merged**